### PR TITLE
Changed recipe to use pypi as the source

### PR DESCRIPTION
**Pull request by mhearne-usgs**
Checklist
* [ ] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,66 +1,51 @@
...
-    - getcsv --help
-    - geteventhist --help
-    - getmags --help
-    - getpager --help
-    - getphases --help
-    - getproduct --help
-    
   imports:
-    - libcomcat
+    - usgs_libcomcat
```
Comment (ocefpaf) [inline recipe/meta.yaml:37]: @mhearne-usgs you still import it as `libcomcat`, no?

Comment (mhearne-usgs) [reply recipe/meta.yaml:37]: Correct, the Python package that is imported is still called libcomcat.

**Comment by mhearne-usgs** [comment]
@conda-forge-admin please rerender

**Comment by mhearne-usgs** [comment]
@ocefpaf I'm confused - no matter what I put in this line:

https://github.com/mhearne-usgs/libcomcat-feedstock/blob/v2020/recipe/meta.yaml#L37

The docker build always says `ModuleNotFoundError: No module named 'usgs_libcomcat'`

**Pull request merged**